tools/esp_secure_cert/custflash_format.py

```diff
@@ -1,9 +1,11 @@
-import sys
 import struct
 import zlib
-from esp_secure_cert.esp_secure_cert_helper import load_private_key, load_certificate
+
 from cryptography.hazmat.primitives import serialization
-from cryptography.hazmat.backends import default_backend
+from esp_secure_cert.esp_secure_cert_helper import (
```

Comment:
Just to note here.
Are you planning to use the custflash_format for your project?
We have already marked the custflash_format as a legacy format.